index.js

```diff
@@ -0,0 +1,64 @@
+// module-2 week-1 - 24 April 2023
+
+import Book from './modules/book.js';
+
+import { DateTime } from './modules/luxon.js';
+
+document.addEventListener('DOMContentLoaded', Book.displayBooks());
+
+const form = document.querySelector('#bookForm');
+
+form.addEventListener('submit', () => {
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+
+  const book = new Book(title, author);
+
+  Book.addBookToList(book);
+  Book.addBook(book);
+});
+
+document.querySelector('#bookOfList').addEventListener('click', (e) => {
+  Book.deleteBook(e.target);
+  Book.removeBook(e.target.previousElementSibling.textContent);
+});
+
+// Get current date and display it
+const date = DateTime.now().toLocaleString(DateTime.DATE_FULL);
+document.querySelector('#date').textContent = date;
```

Comment:
[OPTIONAL] I think if we are using the luxon library it would be better if we also show the time along with the date for the user